superset-frontend/src/dashboard/components/PropertiesModal.tsx

```diff
@@ -199,15 +210,15 @@ class PropertiesModal extends React.PureComponent {
           colorScheme: jsonMetadataObj.color_scheme,
         },
       }));
-      const initialSelectedOwners = dashboard.owners.map(owner => ({
+      const initialSelectedOwners = dashboard.owners.map((owner: Owner) => ({
```

Comment:
woo, i love it when we can pull in types that are already defined! Stuff like this means that typing should hit an inflection point where the typing migrations becomes mostly pulling in types from other places

Reply:
I just found a type in the wild on another PR! Still a long road ahead, no doubt about it, but it's getting better!